### ISSUE TITLE
Enhancement - Remember settings

### ISSUE THREAD

**Issue by DutchGeek** [ThScEo/ForgeOfBot#9]
It would be really nice if I didnt need to enter my account details and the server that I am on every time I start the program.  I would also like it to remember the settings that I had chosen the last time I used the program.

**Comment by ThScEo** [comment]
That will be implemented fast in the next release

**Comment by DutchGeek** [comment]
I tested this out and it is working great!  thank you for this change

**Comment by DutchGeek** [comment]
it would be great if it would also remember the settings for: 
supplies productions
goods production
which options were selected in the quickstart control center
also all settings on each of the tabs

**Comment by ThScEo** [comment]
Yes it will be done in the next release, thank your for your active participation

**Comment by DutchGeek** [comment]
I am happy to help make this an awesome program.  Let me know if there is anything else I can do to help.
